localization/ar.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+$labels = array();
+$labels['internal'] = 'كان البريد داخلياً';
```

Comment:
This is the exact literal translation of the English version; it looks fine to me. if finally you're removing "was" from the English version, so in Arabic this should be the translation. 
```
$labels['internal'] = 'بريد داخلي' ;
```